ops/terraform/services/pipeline/modules/bfd_pipeline_slis/main.tf

```diff
@@ -33,6 +34,14 @@ resource "aws_lambda_function" "this" {
 
   kms_key_arn = local.kms_key_arn
 
+  # Ensures that only _one_ instance of this Lambda can run at any given time. This stops the
+  # possible duplicate submissions to the first available time metric that could otherwise occur
+  # if two instances of this Lambda are invoked close together. This _does_ take from our total
+  # reserved concurrent executions, so we should investigate an even more robust method of stopping
+  # duplicate submissions
+  # TODO: Investigate other methods of stopping duplicate sample submissions to first available metric
```

Comment:
We could create a new card for this, although the more I'm thinking about this the more I feel like this isn't much of a problem. Setting reserved concurrency is a completely valid mechanism for restricting concurrent Lambda executions, and since we're using S3 Event Notifications to invoke the Lambda it's [_asynchronously_ invoked](https://docs.aws.amazon.com/lambda/latest/dg/invocation-async.html), so AWS will automatically queue events if the Lambda is already running.

I can't think of many other means of stopping concurrent execution without creating more headaches for ourselves. I might just remove this `TODO`.